inc/thorvg.h

```diff
@@ -309,7 +309,7 @@ class TVG_EXPORT Paint
      *
      * @return The method used to composite the source object with the target.
      *
-     * @BETA_API
+     * @since 0.4
      */
     CompositeMethod composite(const Paint** target) const noexcept;
```

Comment:
A few issues with composites remain.
I think it is right to open after these are resolved.
https://github.com/Samsung/thorvg/issues/524
https://github.com/Samsung/thorvg/issues/513
https://github.com/Samsung/thorvg/issues/598

Reply:
@JSUYA  Though those are about the setter api which is already in the official, we don't need to rush the release. so removed this one in the candidate.

Reply:
@hermet since we are on the subject: should I add composite getter to the capi?